bgpd/bgp_rpki.c

```diff
@@ -1192,12 +1221,13 @@ DEFUN (show_rpki_prefix_table,
 
 DEFPY (show_rpki_prefix,
        show_rpki_prefix_cmd,
-       "show rpki prefix <A.B.C.D/M|X:X::X:X/M> [(1-4294967295)$asn]",
+       "show rpki prefix <A.B.C.D/M|X:X::X:X/M|(1-4294967295)$by_asn> [(1-4294967295)$asn]",
        SHOW_STR
```

Comment:
I am not a big fan of this shortcut.
I would rather a dedicated command.
I assume problem is that prefix param is mandatory. right?

what about:
"show rpki prefix [<A.B.C.D/M|X:X::X:X/M>] [(1-4294967295)$asn]

Reply:
You mean merging `show_rpki_prefix_table_cmd` and `show_rpki_prefix_cmd` commands?

Reply:
```<A.B.C.D/M|X:X::X:X/M|(1-4294967295)$by_asn>```

arent there any cases where thers is overlap between IPs definition and ASN values?
I did not test it on frr, but on linux shell, you can do `ssh 0`, and it will try to connect to localhost, so I was wondering if there was not a similar case with show rpki prefix 1 ( for 0.0.0.1 IP ?)

Reply:
@pguibert6WIND 
```
spine1-debian-9# sh rpki prefix 1.1.1.1/32
Prefix                                   Prefix Length  Origin-AS
spine1-debian-9# sh rpki prefix 1
```
If you miss CIRD notation then AS number is used instead.

Reply:
What's odd here is the use of `prefix.` Normally you'd think `prefix` means something like "show me everything in the local rpki table related to the prefix I'm entering," but what it means here seems to be "show me all the prefixes related to whatever I enter..." I think that's why it might be causing some confusion. 

Would it make more sense to leave the `prefix` bit out, just making it `sho rpki`? I don't know if this is a good solution, either, though... I understand the confusion, but I don't see a good way to resolve it, so I'm inclined to say "this is okay, so long as its documented well."

Reply:
Or maybe `show rpki as-number <AS_NUMBER>`?